rocky/reports/report_types/definitions.py

```diff
@@ -24,7 +25,7 @@ class Report(ABC):
     template_path: str = "report.html"
 
     def __init__(self, octopoes_api_connector):
-        self.octopoes_api_connector = octopoes_api_connector
+        self.octopoes_api_connector: OctopoesAPIConnector = octopoes_api_connector
```

Comment:
The type hint here is a bit redundant and should be placed next to the argument instead

Reply:
My editor didn't pick it up though. But indeed the argument would be better